action.yml

```diff
@@ -21,10 +21,14 @@ inputs:
     description: 'The comma-separated list of labels for the event. Format: key=value,key2=value2'
     required: false
     default: ''
+  contexts:
+    description: 'Require pipectl >= v0.49.3. The comma-separated list of contexts for the event. Format: key=value,key2=value2'
+    required: false
+    default: ''
```

Comment:
How about adding repository information by default?
IMO, it's helpful that we use repo info without configurations.

Reply:
@Warashi 
Yes, so this is just a support `--contexts` option.
We will think about it later